automated-comments/go/raindrops/multiple_loops.md

```diff
@@ -0,0 +1,3 @@
+Multiple loops are used in this approach.
+Programming is often about finding the simplest working solution.
+Try reducing the amount of loops to one or no loop at all.
```

Comment:
A suggestion I sometimes use: "What would you do if you only had one number to check?"

Reply:
We agree not to use `you` or ask questions as this is a bot submitting the comments.

See https://github.com/exercism/automated-mentoring-support/blob/5b01bae1faced4c771308bab10a4fd6277e2a4d1/docs/comment_copy_guidelines.md

Reply:
"What would be another way to write this program if there were only one number to check?"

Reply:
It is still a question...